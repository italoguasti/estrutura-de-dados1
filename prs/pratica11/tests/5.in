500
4410845 Arthur
3014989 Arthur
7307388 Sophia
8208113 Gabriel
7765899 Davi
9135029 Alice
5036951 Davi
0637738 Gabriel
3482902 Sophia
7826240 Davi
7931228 Sophia
9118918 Sophia
9567964 Davi
9104095 Alice
9608401 Davi
8611869 Alice
2331730 Pedro
6830772 Miguel
3269312 Arthur
4025595 Bernardo
0250848 Arthur
8652814 Davi
8447831 Sophia
4019763 Pedro
4046163 Gabriel
2415605 Pedro
0269711 Helena
6493905 Alice
7883086 Sophia
8009186 Pedro
7913737 Helena
6521856 Gabriel
8554083 Helena
4089321 Arthur
6269611 Pedro
8772793 Arthur
8353258 Davi
5490667 Arthur
5054741 Davi
5002409 Gabriel
4270123 Arthur
3432854 Isabella
3274467 Alice
0848614 Alice
3316699 Miguel
1799932 Davi
5675734 Pedro
3611799 Helena
3392522 Gabriel
7244193 Miguel
5417212 Pedro
8611505 Helena
2103190 Miguel
9352239 Pedro
1082782 Davi
8026926 Bernardo
9118643 Helena
8786446 Alice
0587496 Arthur
1681863 Bernardo
3433264 Alice
8524426 Alice
4511535 Gabriel
1834857 Sophia
8872730 Davi
2317664 Alice
2750042 Bernardo
6159536 Isabella
1608004 Arthur
1035517 Isabella
6403618 Alice
7620505 Pedro
8160589 Bernardo
3120357 Arthur
4335806 Gabriel
8071140 Pedro
2810667 Davi
4600368 Miguel
7257372 Helena
1287673 Davi
7517393 Miguel
0745853 Isabella
0535779 Isabella
5961942 Helena
7157671 Pedro
5394907 Isabella
4038498 Helena
2899733 Arthur
3839100 Miguel
9131478 Sophia
4039799 Helena
6111378 Isabella
0729181 Bernardo
8038397 Davi
8719148 Gabriel
7976013 Isabella
5136302 Gabriel
2362297 Arthur
3774223 Sophia
3852542 Arthur
4475444 Davi
7068075 Pedro
0478982 Sophia
3423431 Isabella
2550735 Helena
4818864 Gabriel
7765774 Pedro
8566816 Davi
0239390 Miguel
8886622 Davi
7590913 Davi
3282036 Isabella
3907011 Gabriel
8977946 Miguel
1072697 Arthur
2133554 Sophia
5628722 Sophia
6777629 Helena
2780782 Bernardo
0288625 Davi
1507487 Miguel
5502940 Pedro
4732104 Alice
3157689 Helena
4288027 Pedro
6636388 Arthur
7661600 Davi
8133205 Isabella
7194980 Sophia
5240332 Davi
6459733 Alice
8152567 Arthur
9274491 Sophia
9835952 Helena
7617423 Pedro
8506068 Pedro
8685605 Helena
1450962 Alice
1762726 Sophia
4831513 Isabella
9074019 Helena
4022136 Davi
9078482 Arthur
3351148 Gabriel
4056363 Alice
0760859 Alice
9780614 Sophia
3974089 Miguel
0423848 Miguel
1613540 Miguel
4153925 Alice
5968821 Alice
0667915 Gabriel
5292828 Pedro
9397617 Miguel
7099201 Bernardo
4375891 Bernardo
8998992 Bernardo
6919904 Sophia
1345333 Arthur
9541447 Alice
3656235 Helena
5167097 Gabriel
9446840 Arthur
8422203 Sophia
3792946 Miguel
4464316 Arthur
2953496 Gabriel
8887141 Gabriel
2775747 Isabella
7152872 Pedro
4461885 Bernardo
1527690 Arthur
2013997 Davi
6501319 Davi
0084401 Davi
0371696 Pedro
3140570 Bernardo
0299995 Helena
4534189 Bernardo
5494155 Bernardo
5829161 Davi
4104931 Arthur
6421625 Miguel
5357561 Arthur
2975163 Pedro
3500662 Arthur
2975041 Arthur
1659832 Bernardo
6907366 Bernardo
3399767 Pedro
3129294 Gabriel
9881121 Alice
5880525 Miguel
7066327 Sophia
5021111 Miguel
5350376 Pedro
3230464 Pedro
9966296 Arthur
2719847 Miguel
1610828 Sophia
2783327 Gabriel
2803394 Arthur
1007454 Arthur
3953792 Pedro
5087989 Pedro
7871732 Helena
1816958 Arthur
0511618 Gabriel
0215773 Alice
7532921 Alice
8914750 Arthur
9494556 Arthur
2295235 Gabriel
7169649 Alice
7724808 Isabella
7454477 Gabriel
9203288 Arthur
3971655 Bernardo
3071490 Isabella
8911053 Gabriel
5637443 Arthur
2084492 Miguel
9726295 Sophia
7637092 Gabriel
9939353 Gabriel
7269769 Miguel
9249218 Alice
1346390 Miguel
1678229 Helena
5720634 Helena
6938770 Helena
5601874 Gabriel
1298582 Davi
1768767 Pedro
2807828 Miguel
4084896 Miguel
8999388 Bernardo
6343702 Miguel
0989420 Helena
5195626 Bernardo
8690599 Gabriel
8639821 Davi
6521269 Bernardo
4389622 Helena
9946962 Miguel
7364443 Davi
5403955 Isabella
3426568 Bernardo
6457200 Gabriel
6147117 Arthur
3346836 Davi
8489014 Gabriel
5678546 Pedro
6645982 Helena
8017056 Alice
2804207 Pedro
7396281 Alice
5497500 Arthur
7126303 Gabriel
9651492 Bernardo
6002222 Isabella
7220951 Gabriel
4300554 Davi
9728560 Gabriel
4799780 Pedro
8143168 Isabella
7895659 Davi
9320616 Davi
8691020 Gabriel
8128597 Davi
8189448 Sophia
3626103 Isabella
0959818 Davi
4688308 Alice
4444449 Helena
6329059 Isabella
6478484 Isabella
7389825 Helena
7332036 Arthur
4958358 Sophia
7859078 Isabella
5909278 Davi
0614277 Alice
4247100 Arthur
1063942 Davi
6802151 Arthur
7954997 Pedro
1604960 Arthur
6998017 Pedro
0310178 Miguel
9338731 Sophia
2074351 Gabriel
5294245 Helena
3967142 Miguel
0116344 Bernardo
8055667 Bernardo
3325724 Miguel
4596253 Sophia
5897192 Sophia
1318983 Davi
1354024 Helena
4906093 Helena
6221675 Miguel
8596962 Davi
3534309 Sophia
8125255 Miguel
1499161 Alice
4727389 Pedro
8865821 Miguel
3371254 Sophia
8504972 Isabella
8483967 Gabriel
8743667 Bernardo
2483023 Bernardo
3035108 Alice
0963294 Gabriel
6661864 Bernardo
2090760 Davi
0565458 Miguel
8423677 Pedro
8579712 Pedro
8396427 Gabriel
9574807 Alice
1008650 Gabriel
2685557 Gabriel
2704906 Gabriel
5954416 Sophia
6082790 Pedro
5194624 Miguel
4798659 Pedro
3480603 Helena
3516602 Helena
5996029 Alice
0067683 Pedro
9740051 Alice
8394414 Pedro
2518112 Miguel
4186879 Isabella
1622951 Miguel
6757038 Sophia
2029439 Helena
2391047 Sophia
2482843 Alice
3788479 Pedro
1737338 Miguel
1325522 Pedro
1986640 Gabriel
5835736 Isabella
4579699 Alice
0551497 Arthur
2096046 Davi
2312658 Bernardo
0091413 Sophia
1074740 Sophia
2425469 Sophia
1732596 Davi
4795720 Gabriel
1140688 Bernardo
0042196 Sophia
3572678 Isabella
2104157 Arthur
3291763 Sophia
5989770 Sophia
6954270 Sophia
7084981 Davi
3019401 Helena
7015632 Davi
7598753 Sophia
4343630 Miguel
8727674 Davi
3578261 Alice
2131799 Isabella
0495999 Isabella
7524002 Pedro
7064493 Sophia
6428513 Isabella
0304529 Helena
5142307 Bernardo
6838422 Alice
1679450 Miguel
5057027 Sophia
6784623 Davi
1477458 Alice
5006806 Alice
4262560 Isabella
2745983 Pedro
5480805 Miguel
0561733 Pedro
7821042 Pedro
0295224 Isabella
7460529 Gabriel
4780406 Davi
7651021 Davi
3430616 Helena
0147436 Alice
2300870 Isabella
7288685 Davi
8102553 Pedro
6092708 Isabella
9428281 Arthur
1681343 Alice
5556080 Alice
6030530 Miguel
0035480 Sophia
6501531 Bernardo
9843151 Alice
8659355 Gabriel
7131581 Alice
6714037 Arthur
1944705 Miguel
0653931 Pedro
7267248 Helena
6421390 Davi
2454522 Alice
6028064 Isabella
4091587 Helena
1390009 Isabella
6264659 Isabella
5808224 Miguel
9294623 Davi
2275176 Pedro
3257070 Davi
1586909 Arthur
8661854 Sophia
7982716 Davi
1368090 Helena
5721565 Miguel
6906540 Davi
4135818 Miguel
6265668 Alice
3456903 Arthur
9942548 Isabella
7709985 Arthur
6632076 Isabella
5140879 Gabriel
7639784 Gabriel
8636695 Davi
9041490 Gabriel
9501762 Alice
8935753 Sophia
1978796 Alice
9294731 Arthur
4260902 Pedro
9872933 Davi
0459610 Helena
1331441 Sophia
4867233 Miguel
2066562 Miguel
2192359 Gabriel
0726829 Arthur
2221009 Isabella
0484009 Alice
5150748 Davi
6836613 Arthur
7595506 Davi
5531110 Arthur
6867511 Alice
8721943 Isabella
4750308 Bernardo
8314475 Isabella
7977484 Miguel
2199875 Bernardo
8039302 Alice
8546459 Sophia
5005964 Arthur
5070371 Helena
2556858 Alice
8305294 Pedro
6264434 Pedro
2672355 Gabriel
7767274 Davi
6365609 Isabella
1748052 Gabriel
6792747 Helena
4756402 Davi
0937764 Gabriel
5114642 Miguel
6542415 Bernardo
6193697 Sophia
9004454 Sophia
7590040 Gabriel
8079915 Gabriel
1187691 Isabella
1072695 Sophia
9300555 Sophia
4612585 Arthur
1478293 Arthur
2678052 Alice
5544322 Arthur
6569394 Davi
6626071 Isabella
1612356 Arthur
7936262 Pedro
6597148 Pedro
3947854 Miguel
3535498 Helena
3519619 Sophia
3655907 Alice
4651795 Pedro
