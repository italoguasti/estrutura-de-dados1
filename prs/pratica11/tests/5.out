0035480 Sophia
0042196 Sophia
0067683 Pedro
0084401 Davi
0091413 Sophia
0116344 Bernardo
0147436 Alice
0215773 Alice
0239390 Miguel
0250848 Arthur
0269711 Helena
0288625 Davi
0295224 Isabella
0299995 Helena
0304529 Helena
0310178 Miguel
0371696 Pedro
0423848 Miguel
0459610 Helena
0478982 Sophia
0484009 Alice
0495999 Isabella
0511618 Gabriel
0535779 Isabella
0551497 Arthur
0561733 Pedro
0565458 Miguel
0587496 Arthur
0614277 Alice
0637738 Gabriel
0653931 Pedro
0667915 Gabriel
0726829 Arthur
0729181 Bernardo
0745853 Isabella
0760859 Alice
0848614 Alice
0937764 Gabriel
0959818 Davi
0963294 Gabriel
0989420 Helena
1007454 Arthur
1008650 Gabriel
1035517 Isabella
1063942 Davi
1072695 Sophia
1072697 Arthur
1074740 Sophia
1082782 Davi
1140688 Bernardo
1187691 Isabella
1287673 Davi
1298582 Davi
1318983 Davi
1325522 Pedro
1331441 Sophia
1345333 Arthur
1346390 Miguel
1354024 Helena
1368090 Helena
1390009 Isabella
1450962 Alice
1477458 Alice
1478293 Arthur
1499161 Alice
1507487 Miguel
1527690 Arthur
1586909 Arthur
1604960 Arthur
1608004 Arthur
1610828 Sophia
1612356 Arthur
1613540 Miguel
1622951 Miguel
1659832 Bernardo
1678229 Helena
1679450 Miguel
1681343 Alice
1681863 Bernardo
1732596 Davi
1737338 Miguel
1748052 Gabriel
1762726 Sophia
1768767 Pedro
1799932 Davi
1816958 Arthur
1834857 Sophia
1944705 Miguel
1978796 Alice
1986640 Gabriel
2013997 Davi
2029439 Helena
2066562 Miguel
2074351 Gabriel
2084492 Miguel
2090760 Davi
2096046 Davi
2103190 Miguel
2104157 Arthur
2131799 Isabella
2133554 Sophia
2192359 Gabriel
2199875 Bernardo
2221009 Isabella
2275176 Pedro
2295235 Gabriel
2300870 Isabella
2312658 Bernardo
2317664 Alice
2331730 Pedro
2362297 Arthur
2391047 Sophia
2415605 Pedro
2425469 Sophia
2454522 Alice
2482843 Alice
2483023 Bernardo
2518112 Miguel
2550735 Helena
2556858 Alice
2672355 Gabriel
2678052 Alice
2685557 Gabriel
2704906 Gabriel
2719847 Miguel
2745983 Pedro
2750042 Bernardo
2775747 Isabella
2780782 Bernardo
2783327 Gabriel
2803394 Arthur
2804207 Pedro
2807828 Miguel
2810667 Davi
2899733 Arthur
2953496 Gabriel
2975041 Arthur
2975163 Pedro
3014989 Arthur
3019401 Helena
3035108 Alice
3071490 Isabella
3120357 Arthur
3129294 Gabriel
3140570 Bernardo
3157689 Helena
3230464 Pedro
3257070 Davi
3269312 Arthur
3274467 Alice
3282036 Isabella
3291763 Sophia
3316699 Miguel
3325724 Miguel
3346836 Davi
3351148 Gabriel
3371254 Sophia
3392522 Gabriel
3399767 Pedro
3423431 Isabella
3426568 Bernardo
3430616 Helena
3432854 Isabella
3433264 Alice
3456903 Arthur
3480603 Helena
3482902 Sophia
3500662 Arthur
3516602 Helena
3519619 Sophia
3534309 Sophia
3535498 Helena
3572678 Isabella
3578261 Alice
3611799 Helena
3626103 Isabella
3655907 Alice
3656235 Helena
3774223 Sophia
3788479 Pedro
3792946 Miguel
3839100 Miguel
3852542 Arthur
3907011 Gabriel
3947854 Miguel
3953792 Pedro
3967142 Miguel
3971655 Bernardo
3974089 Miguel
4019763 Pedro
4022136 Davi
4025595 Bernardo
4038498 Helena
4039799 Helena
4046163 Gabriel
4056363 Alice
4084896 Miguel
4089321 Arthur
4091587 Helena
4104931 Arthur
4135818 Miguel
4153925 Alice
4186879 Isabella
4247100 Arthur
4260902 Pedro
4262560 Isabella
4270123 Arthur
4288027 Pedro
4300554 Davi
4335806 Gabriel
4343630 Miguel
4375891 Bernardo
4389622 Helena
4410845 Arthur
4444449 Helena
4461885 Bernardo
4464316 Arthur
4475444 Davi
4511535 Gabriel
4534189 Bernardo
4579699 Alice
4596253 Sophia
4600368 Miguel
4612585 Arthur
4651795 Pedro
4688308 Alice
4727389 Pedro
4732104 Alice
4750308 Bernardo
4756402 Davi
4780406 Davi
4795720 Gabriel
4798659 Pedro
4799780 Pedro
4818864 Gabriel
4831513 Isabella
4867233 Miguel
4906093 Helena
4958358 Sophia
5002409 Gabriel
5005964 Arthur
5006806 Alice
5021111 Miguel
5036951 Davi
5054741 Davi
5057027 Sophia
5070371 Helena
5087989 Pedro
5114642 Miguel
5136302 Gabriel
5140879 Gabriel
5142307 Bernardo
5150748 Davi
5167097 Gabriel
5194624 Miguel
5195626 Bernardo
5240332 Davi
5292828 Pedro
5294245 Helena
5350376 Pedro
5357561 Arthur
5394907 Isabella
5403955 Isabella
5417212 Pedro
5480805 Miguel
5490667 Arthur
5494155 Bernardo
5497500 Arthur
5502940 Pedro
5531110 Arthur
5544322 Arthur
5556080 Alice
5601874 Gabriel
5628722 Sophia
5637443 Arthur
5675734 Pedro
5678546 Pedro
5720634 Helena
5721565 Miguel
5808224 Miguel
5829161 Davi
5835736 Isabella
5880525 Miguel
5897192 Sophia
5909278 Davi
5954416 Sophia
5961942 Helena
5968821 Alice
5989770 Sophia
5996029 Alice
6002222 Isabella
6028064 Isabella
6030530 Miguel
6082790 Pedro
6092708 Isabella
6111378 Isabella
6147117 Arthur
6159536 Isabella
6193697 Sophia
6221675 Miguel
6264434 Pedro
6264659 Isabella
6265668 Alice
6269611 Pedro
6329059 Isabella
6343702 Miguel
6365609 Isabella
6403618 Alice
6421390 Davi
6421625 Miguel
6428513 Isabella
6457200 Gabriel
6459733 Alice
6478484 Isabella
6493905 Alice
6501319 Davi
6501531 Bernardo
6521269 Bernardo
6521856 Gabriel
6542415 Bernardo
6569394 Davi
6597148 Pedro
6626071 Isabella
6632076 Isabella
6636388 Arthur
6645982 Helena
6661864 Bernardo
6714037 Arthur
6757038 Sophia
6777629 Helena
6784623 Davi
6792747 Helena
6802151 Arthur
6830772 Miguel
6836613 Arthur
6838422 Alice
6867511 Alice
6906540 Davi
6907366 Bernardo
6919904 Sophia
6938770 Helena
6954270 Sophia
6998017 Pedro
7015632 Davi
7064493 Sophia
7066327 Sophia
7068075 Pedro
7084981 Davi
7099201 Bernardo
7126303 Gabriel
7131581 Alice
7152872 Pedro
7157671 Pedro
7169649 Alice
7194980 Sophia
7220951 Gabriel
7244193 Miguel
7257372 Helena
7267248 Helena
7269769 Miguel
7288685 Davi
7307388 Sophia
7332036 Arthur
7364443 Davi
7389825 Helena
7396281 Alice
7454477 Gabriel
7460529 Gabriel
7517393 Miguel
7524002 Pedro
7532921 Alice
7590040 Gabriel
7590913 Davi
7595506 Davi
7598753 Sophia
7617423 Pedro
7620505 Pedro
7637092 Gabriel
7639784 Gabriel
7651021 Davi
7661600 Davi
7709985 Arthur
7724808 Isabella
7765774 Pedro
7765899 Davi
7767274 Davi
7821042 Pedro
7826240 Davi
7859078 Isabella
7871732 Helena
7883086 Sophia
7895659 Davi
7913737 Helena
7931228 Sophia
7936262 Pedro
7954997 Pedro
7976013 Isabella
7977484 Miguel
7982716 Davi
8009186 Pedro
8017056 Alice
8026926 Bernardo
8038397 Davi
8039302 Alice
8055667 Bernardo
8071140 Pedro
8079915 Gabriel
8102553 Pedro
8125255 Miguel
8128597 Davi
8133205 Isabella
8143168 Isabella
8152567 Arthur
8160589 Bernardo
8189448 Sophia
8208113 Gabriel
8305294 Pedro
8314475 Isabella
8353258 Davi
8394414 Pedro
8396427 Gabriel
8422203 Sophia
8423677 Pedro
8447831 Sophia
8483967 Gabriel
8489014 Gabriel
8504972 Isabella
8506068 Pedro
8524426 Alice
8546459 Sophia
8554083 Helena
8566816 Davi
8579712 Pedro
8596962 Davi
8611505 Helena
8611869 Alice
8636695 Davi
8639821 Davi
8652814 Davi
8659355 Gabriel
8661854 Sophia
8685605 Helena
8690599 Gabriel
8691020 Gabriel
8719148 Gabriel
8721943 Isabella
8727674 Davi
8743667 Bernardo
8772793 Arthur
8786446 Alice
8865821 Miguel
8872730 Davi
8886622 Davi
8887141 Gabriel
8911053 Gabriel
8914750 Arthur
8935753 Sophia
8977946 Miguel
8998992 Bernardo
8999388 Bernardo
9004454 Sophia
9041490 Gabriel
9074019 Helena
9078482 Arthur
9104095 Alice
9118643 Helena
9118918 Sophia
9131478 Sophia
9135029 Alice
9203288 Arthur
9249218 Alice
9274491 Sophia
9294623 Davi
9294731 Arthur
9300555 Sophia
9320616 Davi
9338731 Sophia
9352239 Pedro
9397617 Miguel
9428281 Arthur
9446840 Arthur
9494556 Arthur
9501762 Alice
9541447 Alice
9567964 Davi
9574807 Alice
9608401 Davi
9651492 Bernardo
9726295 Sophia
9728560 Gabriel
9740051 Alice
9780614 Sophia
9835952 Helena
9843151 Alice
9872933 Davi
9881121 Alice
9939353 Gabriel
9942548 Isabella
9946962 Miguel
9966296 Arthur
